.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint scripts and CSS
```

Comment:
Can we wrap this back into `test.yaml` to align with the boilerplate?

Reply:
I kind of like the 2 actions - as they address 2 separate issues... but ok. if you like.